### ISSUE TITLE
Run the OVA image with no working DHCP. The printed password is not correct

### ISSUE THREAD

**Issue by gianluca-valentini** [Graylog2/graylog2-images#226]
Graylog 3.0.1

I'm in the scenario described [here](http://docs.graylog.org/en/latest/pages/installation/virtual_machine_appliances.html)

> If you don’t have a working DHCP server for your virtual machine, you will get the error message:
> 
> “Your appliance came up without a configured IP address. Graylog is probably not running correctly!”
> In this case, you have to login and edit /etc/network/interfaces in order to setup a fixed IP address. Then create the file /var/lib/graylog-server/firstboot and reboot.

After reboot the password is shown in the console of the virtual server but it does work.

Gianluca

**Comment by DarthValiant** [comment]
I experienced the same exact issue. I figured out that booting with /var/lib/graylog-server/firstboot present cause successive hashes to be concatenated to the /etc/graylog/server/server.conf root_password_sha2 field with each value present and the newest value leftmost. Each value was separated with a space character.

I generated the hash and was about to edit it in and noticed that it was already there. I cleared the extraneous hashes and rebooted and that works around this problem.
